src/Cryptol/Eval/SBV.hs

```diff
@@ -0,0 +1,764 @@
+       assertSideCondition sym (svNot (svEqual b z)) DivideByZero
+       pure $! svQuot a b  -- TODO! Fix this: see issue #662
+
+  wordMod sym a b =
+    do let z = literalSWord (intSizeOf b) 0
+       assertSideCondition sym (svNot (svEqual b z)) DivideByZero
+       pure $! svRem a b   -- TODO! Fix this: see issue #662
+
+  wordSignedDiv sym a b =
+    do let z = literalSWord (intSizeOf b) 0
+       assertSideCondition sym (svNot (svEqual b z)) DivideByZero
+       pure $! signedQuot a b
+
+  wordSignedMod sym a b =
+    do let z = literalSWord (intSizeOf b) 0
+       assertSideCondition sym (svNot (svEqual b z)) DivideByZero
+       pure $! signedRem a b
+
+  wordExp _ a b = sExp a b
+  wordLg2 _ a = sLg2 a
+
+  wordToInt _ x = pure $! svToInteger x
+  wordFromInt _ w i = pure $! svFromInteger w i
+
+  intEq _ a b = pure $! svEqual a b
+  intLessThan _ a b = pure $! svLessThan a b
+  intGreaterThan _ a b = pure $! svGreaterThan a b
+
+  intPlus  _ a b = pure $! svPlus a b
+  intMinus _ a b = pure $! svMinus a b
+  intMult  _ a b = pure $! svTimes a b
+  intNegate _ a  = pure $! SBV.svUNeg a
+  intLg2 _ a = svLg2 a
+
+  intDiv sym a b =
+    do let z = svInteger KUnbounded 0
+       assertSideCondition sym (svNot (svEqual b z)) DivideByZero
+       pure $! svQuot a b  -- TODO! Fix this: see issue #662
+  intMod sym a b =
+    do let z = svInteger KUnbounded 0
+       assertSideCondition sym (svNot (svEqual b z)) DivideByZero
+       pure $! svRem a b   -- TODO! Fix this: see issue #662
+
+  intExp sym  a b =
+    do let z = svInteger KUnbounded 0
+       assertSideCondition sym (svLessEq z b) NegativeExponent
+       pure $! SBV.svExp a b
+
+  -- NB, we don't do reduction here
+  intToZn _ _m a = pure a
+
+  znToInt _ 0 a = pure a
+  znToInt _ m a =
+    do let m' = svInteger KUnbounded m
+       pure $! svRem a m'
+
+  znEq _ m a b = svDivisible m (SBV.svMinus a b)
+
+  znPlus  _ m a b = sModAdd m a b
+  znMinus _ m a b = sModSub m a b
+  znMult  _ m a b = sModMult m a b
+  znNegate _ m a  = sModNegate m a
+
+-- TODO: implement this properly in SBV using "bv2int"
+svToInteger :: SWord SBV -> SInteger SBV
+svToInteger w =
+  case svAsInteger w of
+    Nothing -> svFromIntegral KUnbounded w
+    Just x  -> svInteger KUnbounded x
+
+-- TODO: implement this properly in SBV using "int2bv"
+svFromInteger :: Integer -> SInteger SBV -> SWord SBV
+svFromInteger 0 _ = literalSWord 0 0
+svFromInteger n i =
+  case svAsInteger i of
+    Nothing -> svFromIntegral (KBounded False (fromInteger n)) i
+    Just x  -> literalSWord (fromInteger n) x
+
+-- Errors ----------------------------------------------------------------------
+
+evalPanic :: String -> [String] -> a
+evalPanic cxt = panic ("[Symbolic]" ++ cxt)
+
+
+-- Primitives ------------------------------------------------------------------
+
+evalPrim :: Ident -> Maybe Value
+evalPrim prim = Map.lookup prim primTable
+
+-- See also Cryptol.Prims.Eval.primTable
```

Comment:
I don't think this module is present any more.  Did you mean `Cryptol.Eval.Concrete.primTable`?